[#263] Build different variants instead of using lambdas from storage (#305)
tezos-commons/baseDAO: docs/specification.md

# Configuration

> BaseDAO is a concrete smart contract, but also a framework to implement various DAOs.

Reviewer: I'd say that `baseDAO` is no longer a concrete smart contract, given that only its variants can be built.

Author: Can you please suggest a good way to describe it at its current state?

Reviewer: Well, I would simply remove the "concrete smart contract" part and leave the fact that it is a framework.

I'd suggest:
```suggestion
BaseDAO is a framework to implement various DAO smart contracts.
```
